sdk/storage/azure-storage-blob/tests/test_blob_client.py

```diff
@@ -170,27 +170,17 @@ def test_create_blob_service_custom_domain(self):
             self.assertTrue(service.primary_endpoint.startswith('https://www.mydomain.com/'))
             self.assertTrue(service.secondary_endpoint.startswith('https://' + self.account_name + '-secondary.blob.core.windows.net'))
 
-    def test_create_service_missing_arguments(self):
-        # Arrange
-
-        for service_type in SERVICES:
-            # Act
-            with self.assertRaises(ValueError):
-                service = service_type(None)
-
-                # Assert
-
```

Comment:
Another thing we should test is what kind of error you get, if any, if you supply the full blob URL to the constructor.
I believe the rest of the URL should be ignored in favour of the specified container/blob names